eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/widgets/AccentColoredProgressBar.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.widgets;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.util.AttributeSet;
+import android.widget.ProgressBar;
+
+import com.schedjoules.eventdiscovery.R;
+import com.schedjoules.eventdiscovery.framework.utils.AttributeColor;
+import com.schedjoules.eventdiscovery.framework.utils.TintedDrawable;
+
+
+/**
+ * {@link ProgressBar} that tints the indeterminate drawable with accent color on pre-Lollipop versions.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class AccentColoredProgressBar extends ProgressBar
+{
+    public AccentColoredProgressBar(Context context, AttributeSet attrs)
+    {
+        super(context, attrs);
+
+        if (Build.VERSION.SDK_INT < 21)
+        {
+            tintIndeterminateDrawable();
+        }
+    }
+
+
+    private void tintIndeterminateDrawable()
+    {
+        Drawable indeterminateDrawable = getIndeterminateDrawable();
+        if (indeterminateDrawable != null)
+        {
+            setIndeterminateDrawable(TintedDrawable.tint(indeterminateDrawable, new AttributeColor(getContext(), R.attr.colorAccent)));
```

Comment:
`TintedDrawable.tint(..)` calls `mutate()`.
It also calls `DrawableCompat.wrap()` first which according to the documentation is need to tint the `Drawable` but I remember it working without that. So I am certain about that, but probably best to keep that, too.

I've renamed the method `tint()` to `tinted()` to better express that the returned object is a changed, different one.